app/package.json

```diff
@@ -24,6 +24,7 @@
     "govuk_frontend_toolkit": "^9.0.1",
     "govuk_template_jinja": "^0.26.0",
     "govuk-elements-sass": "3.1.3",
+    "govuk-frontend": "*",
```

Comment:
Neat.

Something to remember if we ever have to account for v4 and v5 in the review app, but it should be pretty blatant.

Reply:
Thanks @domoscargin

This is the reason for the directory moves 😣 

The top-level project is named [govuk-frontend-repository](https://github.com/alphagov/govuk-frontend/blob/main/package.json#L3)
But the nested package is named [govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/main/package/package.json#L2)

Hence the move to ~`./src`~ → `./package/src`

Otherwise inside `node_modules` we can only see package-relative built files (not the beautiful source)

<img width="317" alt="Built package only" src="https://user-images.githubusercontent.com/415517/232501282-505fb458-006f-4b7b-b56f-211c520d724b.png">

Reply:
> Something to remember if we ever have to account for v4 and v5 in the review app, but it should be pretty blatant.

Yeah we can still install from the registry using:

```
npm install govuk-frontend@4.4.0 --save --workspace app
```

Confirm with `npm ls govuk-frontend` to see the registry package alongside the local package:

```console
├─┬ govuk-frontend-config@ -> ./shared/config
│ └── govuk-frontend@4.5.0 deduped -> ./package
├─┬ govuk-frontend-review@ -> ./app
│ └── govuk-frontend@4.4.0
├─┬ govuk-frontend-webpack-boilerplate@ -> ./docs/examples/webpack
│ └── govuk-frontend@4.5.0 deduped -> ./package
└── govuk-frontend@4.5.0 -> ./package
```

Clever eh?

Bit of a pain involved (this PR) but it sets us up for the future